<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ManasPC" failures="2" tests="50" name="testRunner.TestRun" time="155.859" errors="0" timestamp="2024-04-29T21:40:22 EDT" skipped="0">
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.486"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.321"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="9.578"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.720"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.478"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.522"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.888"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.664"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.964"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.326"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.837"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.318"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="8.760"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.287"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="4.565"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.343"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="7.991"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.490"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="5.779">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1066)
at org.testng.Assert.assertEquals(Assert.java:1106)
at pageObjectModel.LoginPage.CheckErrorMessage(LoginPage.java:278)
at stepDefinitions.LoginStepDef.error_message_please_check_username_password(LoginStepDef.java:178)
at ✽.Error message please check username/password(file:///C:/Numpy_Ninja_2024/Team2_Selenium_Apr2024/src/test/resources/features/01login.feature:103)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.680"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.681"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.843"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="7.317"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.820"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.732"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.437"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.713"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.528"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.496"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.892"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.310"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.294"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.500"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.733"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.679"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.288"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.314"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.698"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.366"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.318"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.481"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.498"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.681"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.303"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="3.294"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.520"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="6.235">
    <failure message="expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1066)
at org.testng.Assert.assertEquals(Assert.java:1106)
at pageObjectModel.LoginPage.CheckErrorMessage(LoginPage.java:278)
at stepDefinitions.LoginStepDef.error_message_please_check_username_password(LoginStepDef.java:178)
at ✽.Error message please check username/password(file:///C:/Numpy_Ninja_2024/Team2_Selenium_Apr2024/src/test/resources/features/01login.feature:97)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="10.666"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.743"/>
  <system-out/>
  <testcase classname="testRunner.TestRun" name="runScenario" time="0.482"/>
  <system-out/>
</testsuite> <!-- testRunner.TestRun -->
